MPF.ExecutionContexts/Redumper/ExecutionContext.cs

```diff
@@ -268,10 +267,10 @@ protected override void SetDefaultParameters(string? drivePath,
             switch (MediaType)
             {
                 case SabreTools.RedumpLib.Data.MediaType.CDROM:
-                    ModeValues = RedumpSystem switch
+                    ModeValue = RedumpSystem switch
```

Comment:
Please keep these as `disc` for now. I understand why you want to change this, but I do not want to make that leap in the code yet.

Reply:
An addendum specifically for the `CDROM` case: There no longer needs to be a switch since both "normal" CD and SACD will have the same value being assigned.